1-Draft/RFCNNNN-Side-By-Side-Support.md

```diff
@@ -0,0 +1,253 @@
+to a Windows PowerShell release can be installed side-by-side with Windows
+PowerShell (note: PowerShell Core 6.x was not a true successor to Windows
+PowerShell, but PowerShell 7 is the successor to both Windows PowerShell 5.x
+and PowerShell Core 6.x). It will even have a separate executable (`pwsh.exe`),
+and can be installed to specific locations. That means users will be able to
+install and use PowerShell 7 side-by-side with other versions of PowerShell,
+which is a great help for users looking for opportunities to move forward.
+There are still some products and/or scenarios that aren't supported in
+PowerShell 7, but this release brings us much, much closer to parity and gives
+us an opportunity to spend more time looking forward than backward.
+
+Unfortunately, according to the [Microsoft blog post about the forthcoming
+PowerShell 7 release](https://devblogs.microsoft.com/powershell/the-next-release-of-powershell-powershell-7/),
+PowerShell 7 will align more closely with the .NET Core support lifecycle,
+which brings in support for LTS (Long Term Servicing) and non-LTS releases.
+Non-LTS (aka Current) releases are supported for three months after a
+subsequent Current release or an LTS release. This means you'll have to update
+those releases regularly, which helps move the community forward. On the other
+hand, LTS releases will be supported either for three years after general
+availability or for a 12 month maintenance period after the next LTS release
+ships, whichever is longer (i.e. if it takes them longer than expected to get
+an LTS release out the door for some reason, the previous LTS release could be
+supported for even longer). This means you'll be able to stick your heels in
+the ground and use an LTS release for a very long time, which risks holding us
+back.
+
+In order to keep the focus on moving forward, the purpose of this proposal is
+to support side-by-side installations of PowerShell such that...:
+
+* ...it is easy to see what versions of PowerShell are installed.
+* ...it is easy to launch a specific installed version of PowerShell from a
+command line.
+* ...having multiple installed versions of PowerShell doesn't cause problems
+for automation, products, or services built for a specific version of PowerShell.
+
+## Motivation
+
+    As a user,
+    I can easily install newer versions of PowerShell side-by-side with old versions,
+    so that I can continue to use products/solutions for specific versions of PowerShell while using newer versions when needed.
+
+    As a user,
+    I can use the command line to see which versions of PowerShell are installed,
+    so that I can know what versions are available to me to use.
+
+    As a user,
+    I can launch the version of PowerShell I need from the command line,
+    so that I can use automation built and tested against specific versions of PowerShell while still being able to use the latest version on the same system.
+
+## User Experience
+
```

Comment:
Does this RFC include Windows? Or is it MacOS/Linux only? Where does WSL fit?

The examples are using a lot of third-party (by which I mean "non-Windows" and "non-PowerShell") commands. I find myself quite confused by this. I know what "apt get" is, but no clue what "brew" or "choco" or etc. do.

Reply:
`brew` is `apt` for macOS and `choco` is `apt` for Windows (3rd party so not available by default).  This RFC needs to work with non-Windows.